.github/workflows/snyk.yml

```diff
@@ -0,0 +1,25 @@
+# ref: https://github.com/snyk/actions/tree/master/gradle-jdk11
+name: snyk for gradle
+on:
+  push:
```

Comment:
The `snyk` and `pr-build` workflow are having the same condition. Can we move this job as part of pr-build.yml workflow? 

As an example, below is the current scenario. 
<img width="1674" alt="Screenshot 2020-12-18 at 1 59 00 PM" src="https://user-images.githubusercontent.com/53209990/102592267-3bf84600-4139-11eb-9389-d05eb746788b.png">

May less workflow makes it easy to look for
1. pr-build (build-and-test)
2. publish (release)